### PR TITLE
LPS-147067

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**

### CODE REVIEW

**Comment by david-gutierrez-mesa** [comment]
ci:forward

**Comment by david-gutierrez-mesa** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-team-extranet/site-initializer-team-extranet-test/src/testFunctional/paths/TeamExtranet.path [hunk 1]
```diff
@@ -29,7 +34,6 @@
 <tr>
 	<td>TITLE_LINK</td>
 	<td>//h3[contains(.,'${key_entryTitle}')]//a</td>
-	<td></td>
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-team-extranet/site-initializer-team-extranet-test/src/testFunctional/paths/TeamExtranet.path:32]: apparently this is necessary for the SF

Comment (catarineaguiar) [reply modules/apps/site-initializer/site-initializer-team-extranet/site-initializer-team-extranet-test/src/testFunctional/paths/TeamExtranet.path:32]: Sorry, my bad. I'm going to fix it.

**Review by catarineaguiar — APPROVED** [review]

**Comment by catarineaguiar** [comment]
ci:close

**Pull request closed without merging**